### ISSUE TITLE
Some of the links are not read with TalkBack on android

### ISSUE THREAD

**Issue by lunars97** [digitalfabrik/integreat-app#2439]
### Describe the Bug

Most of the links are not read/skipped by TalkBack on android native. Also links are not read in a context of a text area. Usually it is read separately when it is clicked.

### Steps to Reproduce

1. Go to 'Categories'
2. Click on 'text areas where the links are located'

### Expected Behavior

It should be read in context of a text, not separately only when the link is clicked.

### Actual Behavior

A link is skipped by TalkBack. Below there is the text area where the link is before the **.** sign. The link inside text area is not read, but **.** and then the chosen text area is read

### Additional Information

![image](https://github.com/digitalfabrik/integreat-app/assets/77638520/9c667ac6-540a-4589-bf5b-a48c6a3cd3ab)

**Comment by steffenkleinle** [comment]
Possibly related to/duplicating #1236.

**Comment by bahaaTuffaha** [comment]
Why the `<a>` tag got that size?
![Image](https://github.com/user-attachments/assets/64659aa8-40e6-489e-b31c-9bd90efcfd4b)
https://integreat.app/augsburg/en/anlaufstellen/the-kultursozialticket-tickets-for-refugees-theatre-for-1

**Comment by bahaaTuffaha** [comment]
> > Why the `<a>` tag got that size?
> > https://integreat.app/augsburg/en/anlaufstellen/the-kultursozialticket-tickets-for-refugees-theatre-for-1
> 
> I think it is just wrongly highlighted by chrome dev tools, can't reproduce the same behavior in firefox. Also, only the "actual" link is clickable, so I think this is fine. Or am I missing anything here?

Yeah you are right.. in firefox it looks fine.

**Comment by bahaaTuffaha** [comment]
Looked at this again and I think this issue has to do with the " . " after the link at :
https://integreat.app/augsburg/de/freizeit-kultur-und-sport/kultursozialticket-tickets-f%C3%BCr-gefl%C3%BCchtete

It somehow affects how the Talkback segments the reading parts... so I tested that paragraph without the dot (with spaces) " . " and it worked..  (I did that using javascript injection) I think this needs to be fixed from CMS.

![Image](https://github.com/user-attachments/assets/39b8bedf-0a9c-493b-bcf9-87c09e365f2e)

**Comment by LeandraH** [comment]
I couldn't reproduce the issue at all, but I was testing on emulated Android 🤷  The reading behavior in the native app and in the browser were exactly the same for me, which is that each HTML element is considered a new item, so it pauses before bold text, if I continue, it reads the bold text and stops, with the next swipe it finishes the paragraph. Same behavior with the link.

@bahaaTuffaha maybe try testing it from the CMS by adding the same page into Testumgebung and playing around with it there?

**Comment by bahaaTuffaha** [comment]
I tested it in CMS and that didn't do any difference 😓

**Comment by LeandraH** [comment]
Rats! Then I guess it is still a problem on our side :(

**Comment by lunars97** [comment]
I think we can close this issue, the whole webview is not consuming right away the focus, a user needs to anyway click on the text in order to get the focus. I remember that i removed the whole focus from the webview because it was not focusing on the button and form...
